testdata/p4tc_samples_outputs/indirect_counter_01_example.template

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/indirect_counter_01_example numtables 1
+
+$TC p4template create action/indirect_counter_01_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/indirect_counter_01_example/ingress/send_nh state active
+
+$TC p4template create action/indirect_counter_01_example/ingress/drop actid 2
+$TC p4template update action/indirect_counter_01_example/ingress/drop state active
+
+$TC p4template create extern/root/Counter extid 1 numinstances 1 tc_acl 0x1136
```

Comment:
2 details here

- we should use the `extid` from the doc (`0x19000001`)
- We should put `has_exec_method` since we need a kernel module for this (this one is also wrong in the doc, we'll fix it there)

```suggestion
$TC p4template create extern/root/Counter extid 0x19000001 numinstances 1 tc_acl 0x1136 has_exec_method
```

Reply:
Updated with new ID